selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/ImportWizardFormTest.java

```diff
@@ -0,0 +1,614 @@
+    importProject.typeDirectoryName("my-webapp");
+    importProject.clickImportBtn();
+    importProject.waitMainFormIsClosed();
+    loader.waitOnClosed();
+    projectExplorer.waitProjectExplorer();
+    projectExplorer.waitItem(currentProjectName);
+    projectExplorer.waitAndSelectItemByName(currentProjectName);
+    projectExplorer.openItemByPath(currentProjectName);
+    loader.waitOnClosed();
+    projectExplorer.waitItemInvisibility(currentProjectName + "/my-lib");
+
+    projectExplorer.expandPathInProjectExplorer(currentProjectName + "/my-webapp/src/main/webapp");
+
+    projectExplorer.openContextMenuByPathSelectedItem(currentProjectName + "/my-webapp");
+    projectExplorer.clickOnItemInContextMenu(GO_INTO);
+    loader.waitOnClosed();
+    projectExplorer.waitVisibilityByName("my-webapp");
+    projectExplorer.waitItemInvisibility(currentProjectName);
+    projectExplorer.openContextMenuByPathSelectedItem(currentProjectName + "/my-webapp");
+    projectExplorer.clickOnItemInContextMenu(GO_BACK);
+    projectExplorer.waitItem(currentProjectName);
+  }
+
+  @Test(priority = 4)
+  public void checkImportProjectInBranchBySshUrl() throws IOException, JsonParseException {
+    currentProjectName = importBranchRepo.getName() + "Ssh";
+    projectExplorer.waitProjectExplorer();
+
+    performImportIntoBranch(importBranchRepo.getSshUrl(), currentProjectName, FIRST_BRANCH);
+    projectExplorer.waitItem(currentProjectName);
+    projectExplorer.waitAndSelectItemByName(currentProjectName);
+    loader.waitOnClosed();
+    menu.runCommand(GIT, BRANCHES);
+    git.waitBranchInTheListWithCoState(FIRST_BRANCH);
+    git.closeBranchesForm();
+    projectExplorer.waitItem(currentProjectName);
+    projectExplorer.expandPathInProjectExplorerAndOpenFile(
+        currentProjectName + "/src/main/java/org.eclipse.qa.examples", "AppController.java");
+    editor.waitActive();
+  }
+
+  @Test(priority = 5)
+  public void checkImportProjectInBranchByHttpsUrl() throws IOException, JsonParseException {
+    currentProjectName = importBranchRepo.getName() + "Https";
+    projectExplorer.waitProjectExplorer();
+
+    performImportIntoBranch(importBranchRepo.getHtmlUrl(), currentProjectName, SECOND_BRANCH);
+    projectExplorer.waitItem(currentProjectName);
+    projectExplorer.waitAndSelectItemByName(currentProjectName);
+    loader.waitOnClosed();
+    menu.runCommand(GIT, BRANCHES);
+    git.waitBranchInTheListWithCoState(SECOND_BRANCH);
+    git.closeBranchesForm();
+
+    projectExplorer.waitItem(currentProjectName);
+    projectExplorer.expandPathInProjectExplorerAndOpenFile(
+        currentProjectName + "/src/main/java/org.eclipse.qa.examples", "AppController.java");
+    editor.waitActive();
+  }
+
+  @Test(priority = 6)
+  public void checkImportProjectInBranchFromGitHub() throws Exception {
+    currentProjectName = importBranchRepo.getName();
+    projectExplorer.waitProjectExplorer();
+
+    importIntoBranchFromGitHub();
```

Comment:
It's hard to understand test scenario when we have just list of 27 commands.
Please divide separate steps with empty rows and comments.